packages/components/Menu/src/MenuTrigger/MenuTrigger.types.ts

```diff
@@ -1,7 +1,12 @@
+import * as React from 'react';
 import { InteractionEvent, PressablePropsExtended } from '@fluentui-react-native/interactive-hooks';
 
 export const menuTriggerName = 'MenuTrigger';
 
+export interface MenuTriggerProps {
+  children?: React.ReactElement | null;
+}
```

Comment:
We need to add this so that we can access the inner render? Is there some other type we can use that `MenuTriggerProps` can extend so we don't have to explicitly do this?

Reply:
MenuTrigger doesn't have an inner render. An explicit prop type was added because it was needed for the render function, since it's now using a regular FC.

To be clear our MenuTrigger doesn't have any props outside of children. All of its stuff comes from the context or the child.

Reply:
So could we just set it's type to `React.FunctionComponent` or `React.FC`?

Reply:
We can, although it's less limiting. Specifying the children prop like this restricts the child to always be one element (instead of say accepting multiple children or an array or something).

I can change it to use FunctionComponent<{}> though

Reply:
If it's better than `FunctionComponent<{}>` because it only supports one child, then that makes sense. I'd maybe ask to make that it's own type to make that clear and then have `MenuTriggerProps` extend that. But.. not a strong opinion. Do what feels best.

Reply:
Yeah keeping it as it is would also keep the code cleaner since it doesn't have to check for or deal with string or number children.

I'm trying to extract the type out into its own type but having a hard time getting TS to cooperate. I'll try a bit more but if I can't get it to work I'll probably merge as is, sorry.

Reply:
And I got it to work. let me know if you were expecting something different.